android/app/src/main/java/com/inaturalistreactnative/MainApplication.java

```diff
@@ -7,23 +7,25 @@
 import com.facebook.react.ReactInstanceManager;
 import com.facebook.react.ReactNativeHost;
 import com.facebook.react.ReactPackage;
+import com.facebook.react.config.ReactFeatureFlags;
 import com.facebook.soloader.SoLoader;
+import com.inaturalistreactnative.newarchitecture.MainApplicationReactNativeHost;
 import java.lang.reflect.InvocationTargetException;
 import java.util.List;
 import com.lugg.ReactNativeConfig.ReactNativeConfigPackage;
 import com.facebook.react.bridge.JSIModulePackage;
-import com.swmansion.reanimated.ReanimatedJSIModulePackage;
+// import com.swmansion.reanimated.ReanimatedJSIModulePackage;
```

Comment:
If we don't need the stuff you've commented out in this file, would you please just remove those lines?

Reply:
Have removed the commented out code